week2ShortAnswers

```diff
@@ -0,0 +1,100 @@
+
+You can manually add another view by copying the current instance of the preview and you can change and modify the view by choosing another device, orientation, and dark/regular view. 
+
+i) Below is an image of the Canvas from Xcode. The Canvas is in selectable mode. Can you explain why the red background does not cover the entire button area?
+￼
+
+Because a view only takes up as much as space as needed. With this error you would want to ensure to implement the frame before the background.
+
+j) Modifier padding(10) adds padding to the view's top, bottom, left, and right sides. How could a padding of 10 be added to only the left and right sides of the view? The answer for this question should be a short section of code.
+
+.padding(.horizontal)
+
+k) Provide two reasons why you would want to extract views.
+
+One of the main reasons to extract code is to keep the code organized, modular, and reusable. 
+This makes it easier to make changes in the app. You only need to go to one view to make the changes
+
+l) How can you determine, in code, if the device is in Portrait or Landscape mode?
+
+@Environment(\.verticalSizeClass) var verticalSizeClass
+@Environment(\.horizontalSizeClass) var horizantalSizeClass
+
+If verticalSizeClass == .regular &&
+    horizontalSizeClass == .compact {
+Code
+}
+
+m) What is a literal value?
+
+A literal value is any number without a decimal point. 
+
+o) What are the safe areas?
+
+It’s the top of the screen with the battery indicator. Not somewhere where you would want to put any writing. But if you want to ignore the safe area you can use “.ignoreSafeArea()”
+
+p) This line of code was in the lesson on animation. Can you state in English what the line means?
+.frame(width: wideShapes ? 200 : 100)
+If the width is wideShaps is true use 200 otherwise use 100
+
+q) Describe the two transitions you were introduced to in this week’s lesson.
+
+There was ease in, ease out. That’s where the view is slowly moved into the screen and moved out slowly.
+There was also the scale transition, where the view shrinks down and scales up. 
+
+r) In Bullseye, the Game struct is what type of object?
+
+It is a model
+
+s) What are SFSymbols?
+
+Those are shapes that are commonly used in app development. Like the redo symbol, the three little lines for menu. You can access these images with the SFSymbols app and in code by using systemName
+
+t) What is the difference between “step into” and “step over " in the debugger?”
+
+When using step into you are stepping into the method letting it run, if you step over, your skipping that line of code
+
+u) Name some items you would place in the Asset Catalog (Assets.)
+
+Background colors, stroke colors. You would put colors that are used in the app in the asset catalog. 
+
+v) How do you change the Display Name of your app?
+
+You select the main folder, the very top, and you go to the general tab —> identify —> display name
```

Comment:
You can change the Display Name of your app in Xcode by navigating to your target's "General" settings. There's a field labeled "Display Name" where you can input the desired name. This is what will be shown on the home screen under your app's icon.